applications/minotari_node/src/config.rs

```diff
@@ -142,6 +142,8 @@ pub struct BaseNodeConfig {
     pub state_machine: BaseNodeStateMachineConfig,
     /// Obscure GRPC error responses
     pub report_grpc_error: bool,
+    // Interval to check if the base node is still in sync with the network
+    pub tari_pulse_interval: Option<Duration>,
```

Comment:
I dont think serde does handle this option here correct. 
If it does I stand corrected. Please test it, but if it does not, just make it a simple duration
But please add an example here to the config file about the option with the default in: https://github.com/tari-project/tari/blob/development/common/config/presets/c_base_node_c.toml
All the other options are their with their defaults. And they are commented out, but give users an indication what they can set.